pages/common/rip.md

```diff
@@ -0,0 +1,28 @@
+# rip
+
+> Remove files or directories by sending them to the graveyard, allowing for them to be recovered.
+> More information: <https://github.com/nivekuil/rip>.
+
+- Remove files or directories from specified locations and place them in the graveyard:
+
+`rip {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
+
+- Interactively remove files or directories, with a prompt before every removal:
+
+`rip -i {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
```

Comment:
We prefer long arguments if available
```suggestion
`rip --inspect {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
```

Reply:
@superatomic this new description is not clear about the `prompt`.

Reply:
Ah yes, you're right about that. How about this:
```markdown
- Interactively [i]nspect files or directories and prompt for confirmation before removing them:

`rip -i {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
```

Reply:
Okay, that's fair; I'll make the changes. Should I also change the description to my new suggestion?

Reply:
I like them both equally. Take whichever you prefer.

Reply:
Got it! I think I'm going to keep it as is.